internal/cache/lru/lru.go

```diff
@@ -267,3 +268,11 @@ func (c *Cache) UpdateWithoutChangingOrder(
 
 	return nil
 }
+
+func (c *Cache) EraseEntriesWithGivenPrefix(prefix string) {
+	for key := range c.index {
+		if strings.HasPrefix(key, prefix) {
+			c.Erase(key)
```

Comment:
Is it not complaining that you are not handling the return type?

Reply:
It's not throwing any errors. It's only returning deleted entries that we don't need. We're also not getting any warnings about this, so I ignore catching return type.

Reply:
SG